src/System.Private.CoreLib/src/System/Diagnostics/Contracts/Contracts.cs

```diff
@@ -629,7 +629,7 @@ public static bool ForAll(int fromInclusive, int toExclusive, Predicate<int> pre
 #if INSIDE_CLR
                 throw new ArgumentException(SR.Argument_ToExclusiveLessThanFromExclusive);
 #else
-                throw new ArgumentException("fromInclusive must be less than or equal to toExclusive.");
+                throw new ArgumentException($"{nameof(fromInclusive)} must be less than or equal to {nameof(toExclusive)}.");
 #endif
```

Comment:
Can't we just add Argument_ToExclusiveLessThanFromExclusive as a string resource and skip the whole INSIDE_CLR check?